test/isochrone.cc

```diff
@@ -101,23 +100,23 @@ TEST(Isochrones, Basic) {
7],[5.060321,52.069593],[5.054919,52.070937],[5.060329,52.078937],[5.059321,52.08115],[5.057797,52.078461],[5.054992,52.077937],[5.05566,52.074598],[5.052834,52.07445],[5.053321,52.077735],[5.050321,52.072423],[5.051237,52.075937],[5.047708,52.078324],[5.048321,52.081429],[5.043259,52.075937],[5.042771,52.072487],[5.035321,52.073258],[5.034141,52.077937],[5.029369,52.075937],[5.030321,52.07481],[5.022061,52.075197],[5.021982,52.073598],[5.025321,52.073406],[5.026952,52.071568],[5.025209,52.067937],[5.029839,52.071419],[5.033321,52.071573],[5.063321,52.066206],[5.064688,52.063304],[5.067613,52.061937],[5.065639,52.057937],[5.06776,52.057376],[5.068263,52.054879],[5.069968,52.05829],[5.073608,52.058224],[5.071012,52.057246],[5.06975,52.051508],[5.066428,52.052044],[5.066186,52.051072],[5.072978,52.046594],[5.089321,52.046308],[5.090321,52.047264],[5.093321,52.046314],[5.094321,52.048197],[5.098321,52.046251],[5.101182,52.051076],[5.104099,52.050159],[5.10757,52.051186],[5.111485,52.050101],[5.112321,52.046219],[5.113321,52.049186],[5.114893,52.046509],[5.118321,52.046164],[5.120321,52.049163],[5.124321,52.048191],[5.141321,52.050285],[5.142321,52.048204],[5.145104,52.052154],[5.155792,52.051466],[5.156139,52.054119],[5.159029,52.054937],[5.155145,52.055761],[5.155123,52.057135],[5.160004,52.058937],[5.157321,52.059628],[5.153088,52.057704],[5.151701,52.063937],[5.149182,52.063798],[5.149126,52.065937],[5.152321,52.068117],[5.156739,52.064355],[5.159112,52.064937],[5.158932,52.066548],[5.151959,52.070937],[5.161802,52.079456],[5.163074,52.091937],[5.159648,52.093264],[5.153648,52.102937],[5.161819,52.104937],[5.154033,52.104649],[5.148684,52.1133],[5.149527,52.117937],[5.146321,52.122254],[5.144821,52.119937],[5.147974,52.113284],[5.142321,52.110978],[5.143963,52.108937],[5.141321,52.106324],[5.140321,52.108765],[5.133321,52.105371],[5.134213,52.106937],[5.131321,52.107349],[5.129321,52.104532],[5.115321,52.106504],[5.111321,52.104524],[5.110531,52.108147],[5.107983,52.107599],[5.107321,52.109955],[5.103321,52.106677],[5.100321,52.107739],[5.097321,52.105768],[5.09847,52.109086],[5.094321,52.107837],[5.092321,52.110126],[5.080321,52.112602],[5.081321,52.107534],[5.084321,52.106528],[5.087321,52.107447],[5.089651,52.105607],[5.085321,52.103602],[5.082408,52.106024],[5.077321,52.106621],[5.077321,52.104335],[5.081656,52.102272],[5.082616,52.099641],[5.078321,52.10046],[5.077321,52.098424],[5.071321,52.09879],[5.070321,52.097363],[5.055535,52.110937],[5.059321,52.112586],[5.063239,52.110019],[5.057779,52.115937],[5.058321,52.117587],[5.062832,52.117937],[5.062321,52.119025],[5.057532,52.119937],[5.058321,52.118287],[5.05593,52.119546],[5.047291,52.116937],[5.049872,52.114937],[5.055683,52.115299],[5.053596,52.111662],[5.051321,52.111621],[5.038321,52.116248],[5.035835,52.118451],[5.037321,52.121382],[5.039321,52.119996],[5.041321,52.120514],[5.038507,52.122937],[5.040321,52.125974]],"type":"LineString"},"type":"Feature"}],"type":"FeatureCollection"})";
```

Comment:
I checked iou metric(intersection over union) of the polygons: it is about 0.9999. 
In details a few points shifted slightly(1e-5-1e-6 delta) so I just set new `expected_value`